test/pytest/test_bridge.py

```diff
@@ -78,6 +78,7 @@ async def test_echo(self):
         await self.transport.assert_msg('', command='done', channel=echo)
         await self.transport.assert_msg('', command='close', channel=echo)
 
+    @unittest.skip
```

Comment:
`@todo` perhaps? Skipping without a reason or comment feels like we're going to forget about it.

Reply:
I filed https://issues.redhat.com/browse/COCKPIT-983 to track this.